### PR TITLE
Supporting writeReport options

### PR DESCRIPTION
**Pull request by zsoltvad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Add the plugin, reporter and reporter configuration in your `karma.conf.js`.
   reporters: ['progress', 'karma-remap-istanbul'],
   remapIstanbulReporter: {
     remapOptions: {}, //additional remap options
+    writeReportOptions: {}, //additional writeReport options
```
Comment (mattlewis92) [inline README.md]: Can you rename it from `writeReportOptions` to `reportOptions` as this is what remap-istanbul calls the arg in their docs: https://github.com/SitePen/remap-istanbul/blob/master/src/writeReport.js#L22

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "karma-remap-istanbul",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (mattlewis92) [inline package.json]: Can you remove the version bump, as this is a feature it'll actually be 0.5.0

**Review by mattlewis92 — CHANGES_REQUESTED** [review]

**Comment by mattlewis92** [comment]
Added a couple of minor changes, other than that LGTM!

**Comment by zsoltvad** [comment]
Requested changes are done.

**Review by mattlewis92 — APPROVED** [review]

**Comment by mattlewis92** [comment]
Nice one, thanks!

**Pull request merged**